src/uct/api/uct.h

```diff
@@ -1148,8 +1148,8 @@ ucs_status_t uct_ep_check(const uct_ep_h ep, unsigned flags,
  * @ingroup UCT_RESOURCE
  * @brief Obtain a notification file descriptor for polling.
  *
- * Only interfaces supporting the @ref UCT_IFACE_FLAG_EVENT_FD implement this
- * function.
+ * Only interfaces supporting any of the UCT_IFACE_FLAG_EVENT* flags implement
```

Comment:
Different grammar would be more clear:
"Only interfaces that support at least one of the UCT_IFACE_FLAG_EVENT* flags will implement"